lib/generator/exercise_case.rb

```diff
@@ -27,5 +27,16 @@ def method_missing(sym, *args, &block)
     def respond_to?(sym, include_private = false)
       canonical.respond_to?(sym) || super
     end
+
+    protected
+
+    def underscore(target)
+      target.underscore
```

Comment:
I think it's best if we still use the refinement within ```ExerciseCase```. If we don't, we will have to define two separate ```underscore``` methods, one for ```Fixnum``` and one for ```String```, or define a single ```underscore``` method that switches on the type of the argument, but I really dislike that option.